client/blocks/reader-post-options-menu/index.jsx

```diff
@@ -215,13 +215,13 @@ class ReaderPostOptionsMenu extends React.Component {
 
 					{ this.props.showVisitPost && post.URL && (
 						<PopoverMenuItem onClick={ this.visitPost } icon="external">
-							{ translate( 'Visit Post' ) }
+							{ translate( 'Visit post' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.